core/src/cats/sql_create.cc

```diff
@@ -483,7 +483,7 @@ bool BareosDb::CreateMediaRecord(JobControlRecord* jcr, MediaDbRecord* mr)
            "UPDATE Media SET LabelDate='%s' "
            "WHERE MediaId=%d",
            dt, mr->MediaId);
-      retval = UPDATE_DB(jcr, cmd);
+      retval = UPDATE_DB(jcr, cmd) > 0 ? true : false;
```

Comment:
`x > 0` already is a boolean expression, no need to convert it again.
```suggestion
      retval = UPDATE_DB(jcr, cmd) > 0;
```